packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AbstractModal from 'terra-abstract-modal';
+import Button from 'terra-button';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import { elementType } from 'airbnb-prop-types';
+import styles from './NotificationDialog.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Title of the notification-dialog.
+   */
+  title: PropTypes.string,
+  /**
+   * Message of the notification-dialog.
+   */
+  message: PropTypes.string,
+  /**
+   * Array of buttons. Restricted to terra-button.
+   */
+  actions: PropTypes.arrayOf(elementType(Button)),
+  /**
+   * Toggle to show notification-dialog or not.
+   */
+  showNotificationDialog: PropTypes.bool.isRequired,
```

Comment:
I would gather a few opinions on this prop name. Many of our components use `isOpen` to address the open / closed state of a component.

Reply:
changed it to `isOpen` to maintain consistency.